packages/opentelemetry-instrumentation-http/src/utils.ts

```diff
@@ -218,7 +218,10 @@ export const getRequestInfo = (
       Object.assign(optionsParsed, extraOptions);
     }
   } else {
-    optionsParsed = Object.assign({}, options);
+    optionsParsed = Object.assign(
+      { protocol: options.host ? 'http:' : undefined },
```

Comment:
FYI, i needed to check a required property because we have some tests that pass empty object option to set a protocol.